doc/docs/2.3.x/how-tos/jupyterlab-extension/index.md

```diff
@@ -1,173 +1,115 @@
+USER root
 
-If you are using our pre-built image:
+# This is the directory files will be mounted to, mirroring how pipelines are run. 
+RUN mkdir -p /pfs 
 
-- Run the script below:
+# If you are not using "jovyan" as your notebook user, replace the user here. 
+RUN chown $NB_USER /pfs
 
-    ```shell
-    docker run -it -p 8888:8888 -e GRANT_SUDO=yes --user root --device /dev/fuse --privileged --entrypoint /opt/conda/bin/jupyter pachyderm/notebooks-user:{{ config.jupyterlab_extension_image_tag }} lab --allow-root 
-    ```
+# Fuse is a requirement for the mount extension 
+RUN apt-get clean && RUN apt-get update && apt-get -y install curl fuse 
 
-- Click on the link provided in the stdout of your terminal to run JupyterLab in a browser.
+# Install Pachctl - Set the version of Pachctl that matches your cluster deployment. 
+RUN curl -f -o pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v${PACHCTL_VERSION}/pachctl_${PACHCTL_VERSION}_amd64.deb 
+RUN dpkg -i pachctl.deb
 
-- Jump to the [`Connect Your JupyterLab Extension To Your Pachyderm Cluster`](#connect-the-extension-to-your-pachyderm-cluster) section.
+# This sets the user back to the notebook user account (i.e., Jovyan) 
+USER $NB_UID
 
-Replace the image name with your own image otherwise.
+# Replace the version here with the version of the extension you would like to install from https://pypi.org/project/jupyterlab-pachyderm/ 
+RUN pip install jupyterlab-pachyderm==<version> 
+```
 
-### Adding The Extension To Your JupyterHub Deployment With Helm
+Then, [build, tag, and push your image](../developer-workflow/working-with-pipelines/#step-2-build-your-docker-image).
 
-!!! Info
-    Find the complete installation instructions of JupyterHub on Kubernetes in [Jupyterhub for Kubernetes documentation](https://zero-to-jupyterhub.readthedocs.io/en/latest/#setup-jupyterhub){target=_blank}.
+### Install to JupyterHub With Helm
 
-- As a FUSE requirement, add the following to your **Jupyterhub helm chart values.YAML** file to enable root in the `singleuser` containers or use our default [`jupyterhub-ext-values.yaml`](https://github.com/pachyderm/pachyderm/blob/{{ config.pach_branch }}/etc/helm/examples/jupyterhub-ext-values.yaml){target=_blank}:
+#### Option 1: Pachyderm Defaults
 
-    !!! Note
-        Update the fields `singleuser.image.name` and `singleuser.image.tag` to match your user image or leave Pachyderm's default image `pachyderm/notebooks-user:{{ config.jupyterlab_extension_image_tag }}`.
+1. Open a terminal.
+2. Run the following:
+   ```sh
+   helm upgrade --cleanup-on-fail \
+   --install jupyter jupyterhub/jupyterhub \
+   --values https://raw.githubusercontent.com/pachyderm/pachyderm/{{ config.pach_branch }}/etc/helm/examples/jupyterhub-ext-values.yaml
+   ```
+#### Option 2: Custom
 
+1. Add the following to your Jupyterhub helm chart `values.YAML` file:
```

Comment:
@chainlink For users that install to JH, do we want them to exclusively use the sidecar setup or is the original approach of a combined privileged image still desirable too?